src/components/TransactionList/TransactionRow/TransactionRowTs.ts

```diff
@@ -456,27 +466,19 @@ export class TransactionRowTs extends Vue {
         return this.explorerBaseUrl.replace(/\/+$/, '') + '/transactions/' + this.transaction.transactionInfo.hash;
     }
 
-    public get deadline() {
-        return this.transaction.deadline
-            .toLocalDateTime(this.networkConfiguration.epochAdjustment)
-            .format(DateTimeFormatter.ofPattern('yyyy-MM-dd HH:mm:ss'));
-    }
     public get date() {
         if (this.transaction instanceof AggregateTransaction) {
-            return this.transaction.deadline
-                .toLocalDateTime(this.networkConfiguration.epochAdjustment)
-                .minusHours(48)
-                .format(DateTimeFormatter.ofPattern('yyyy-MM-dd HH:mm:ss'));
+            return moment(
+                String(this.transaction.deadline.toLocalDateTime(this.networkConfiguration.epochAdjustment).minusHours(48)),
```

Comment:
seem there is a few repeated codes.
It will be great, create a method in the `TimeHelpers` class.